Commands/add-blacklist.js

```diff
@@ -0,0 +1,23 @@
+const { DatabaseService } = require('../services/database.service');
+const { prefix, adminRole } = require('../config.json');
+
+module.exports = async function (message) {
+	const dbService = new DatabaseService();
+	if (!message.member.roles.cache.some((role) => role.name === adminRole)) {
+		message.reply('You are not authorized to do that');
+	}
+	let args = message.content.slice(prefix.length).trim().split(' ');
+	args.shift();
+	args = args.map((el) => el.toLowerCase());
```

Comment:
every argument is converted to lowercase, so the bot cant be joked by typing blacklisted words in caps or somehow other